README.md

```diff
@@ -58,59 +52,72 @@ Before you do so, you should check out our contributing guidelines in the
 `CONTRIBUTING.md` file, to make sure it's as easy as possible for us to accept
 your contribution.
 
-## How do I hack on Felix?
+## How do I build Felix?
+
+Felix uses Docker for builds.  We develop on Ubuntu 16.04 but other Linux
+distributions should work (we have not tries OS X).  To build felix, you will need to install:
+
+- Docker 1.12+
+- GNU make.
+- Plenty of disk space (since the builds need to use some heavyweight 
+  full-OS containers in order to build debs and RPMs).
+
+Then, run `make felix-docker-image`, for example, to build the `calico/felix`
+container or `make help` for other options.
```

Comment:
`rkt` does support docker images but they're not its native format.  Might be nice to add a `make felix-rkt-image` that does spit out the tar-based rkt image so I think the terminology is correct.